framework/PostProcessors/ParetoFrontierPostProcessor.py

```diff
@@ -100,11 +112,18 @@ def run(self, inputIn):
     coordinates = np.zeros(1,dtype=int)
     for index,elem in enumerate(sortedData[self.costID].values):
       if (index>1) and (sortedData[self.valueID].values[index]>sortedData[self.valueID].values[coordinates[-1]]):
+        # the point at index is part of the pareto frontier
+        # it is now needed to check if it satisfies the cost and/or value limits
         coordinates = np.append(coordinates,index)
 
-    selection = sortedData.isel(RAVEN_sample_ID=coordinates).to_array().values
-    paretoFrontierData = np.transpose(selection)
+    selection = sortedData.isel(RAVEN_sample_ID=coordinates)
+    if self.valueLimit is not None:
+      selection = selection.where(selection[self.valueID]>self.valueLimit)
```

Comment:
Do you want to include self.valueLimit in your selection, I mean ```selection[self.valueID]>=self.valueLimit```

Reply:
added